### ISSUE TITLE
Stats Endpoint on Documentation

### ISSUE THREAD

**Issue by UghThatGuyAgain** [BenFradet/RiotSharp#531]
The documentation still has the stats endpoint marked down. Is it still just out of date?

**Comment by BenFradet** [comment]
which documentation do you mean?

**Comment by UghThatGuyAgain** [comment]
Specifically, I'm referring to [this](http://benfradet.github.io/RiotSharp/api/RiotSharp.StatsEndpoint.html) (as well as it's enum counterpart). Things like AggregatedStat, ChampionStats, etc have been removed from the API completely, as the endpoint was removed.

**Comment by BenFradet** [comment]
yes, I haven't had time to get to #489 yet, maybe this week :+1:
